crates/task-impls/src/quorum_vote.rs

```diff
@@ -418,11 +412,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumVoteTaskState<TYPES, I
             return;
         }
 
-        debug!(
```

Comment:
I added these two logs when debugging, but the event is mostly `None` so they're just noise.